src/EventStore.Client.PersistentSubscriptions/EventStorePersistentSubscriptionsClient.Info.cs

```diff
@@ -0,0 +1,58 @@
+using System.Threading;
+using System.Threading.Tasks;
+using EventStore.Client.PersistentSubscriptions;
+
+#nullable enable
+namespace EventStore.Client {
+	partial class EventStorePersistentSubscriptionsClient {
+		/// <summary>
+		/// Gets the status of a persistent subscription to $all
+		/// </summary>
+		/// <param name="groupName"></param>
+		/// <param name="userCredentials"></param>
+		/// <param name="cancellationToken"></param>
+		/// <returns></returns>
+		public async Task<PersistentSubscriptionDetail> GetInfoForAllAsync(string groupName,
+			UserCredentials? userCredentials = null, CancellationToken cancellationToken = default) {
+
+			var req = new GetInfoReq() {
+				Options = new GetInfoReq.Types.Options{
+					GroupName = groupName,
+					All = new Empty()
+				}
+			};
+
+			return await GetInfoInternalAsync(req, userCredentials, cancellationToken).ConfigureAwait(false);
+		}
+
+		/// <summary>
+		/// Gets the status of a persistent subscription to a stream
+		/// </summary>
+		/// <param name="groupName"></param>
+		/// <param name="stream"></param>
+		/// <param name="userCredentials"></param>
+		/// <param name="cancellationToken"></param>
+		/// <returns></returns>
+		public async Task<PersistentSubscriptionDetail> GetInfoForStreamAsync(string stream, string groupName,
+			UserCredentials? userCredentials = null, CancellationToken cancellationToken = default) {
+
+			var req = new GetInfoReq() {
+				Options = new GetInfoReq.Types.Options {
+					GroupName = groupName,
+					StreamIdentifier = stream
+				}
+			};
+			return await GetInfoInternalAsync(req, userCredentials, cancellationToken).ConfigureAwait(false);
+		}
+		
+		private async Task<PersistentSubscriptionDetail> GetInfoInternalAsync(GetInfoReq req,
+			UserCredentials? userCredentials, CancellationToken cancellationToken) {
+
+			var result = await new PersistentSubscriptions.PersistentSubscriptions.PersistentSubscriptionsClient(
```

Comment:
**QUESTION:** Shouldn't we have here an HTTP fallback?

Reply:
Done.